apis/nucleus/src/viz.js

```diff
@@ -119,14 +122,20 @@ export default function viz({ model, halo, initialError, onDestroy = async () =>
      * viz.convertTo('barChart');
      */
     async convertTo(newType, forceUpdate = true) {
```

Comment:
I think we'd want to update the API to be clearer as well, so that you more easily can pick between permanent conversion and session based.
+ make sure there is a test and example somewhere.